### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️/❌
| Fixed issues  | #11

### CODE REVIEW

**Comment by samdark** [comment]
Seems linter isn't happy.

**Comment by rustamwin** [comment]
> Seems linter isn't happy.

Yes, I'm fixing errors...

File: src/app/pages/index/components/list-requests/list-requests.component.ts [hunk 1]
```diff
@@ -30,21 +42,19 @@ export class ListRequestsComponent implements OnInit, AfterViewInit {
...
-    return (Number(memory / 1048576)).toFixed(3) + ' MB';
+    return `${Number(memory / 1048576).toFixed(3)} MB`;
   }
 
   formatTime(time: number): string {
-    return (Number(time * 1000)).toFixed(0) + ' ms';
+    return `${Number(time * 1000).toFixed(0)} ms`;
   }
 
   formatID(id: string) {
```
Comment (Julian-B90) [inline src/app/pages/index/components/list-requests/list-requests.component.ts]:
```suggestion
  formatID(id: string): string {
```

File: src/app/pages/index/components/list-requests/list-requests.component.ts [hunk 2]
```diff
@@ -30,21 +42,19 @@ export class ListRequestsComponent implements OnInit, AfterViewInit {
 
   getDebugsList() {
```
Comment (Julian-B90) [inline src/app/pages/index/components/list-requests/list-requests.component.ts]:
```suggestion
  getDebugsList(): void {
```

**Pull request merged**